### DISCUSSION TITLE
How to increase the size of the foliate's "translation balloons"?

### DISCUSSION THREAD

**Discussion by i-m-a-r** [johnfactotum/foliate#845] [Q&A] [answered]
I would like to know if it is possible to increase in size, "the translation balloons" of the foliate, i find this function very interesting, but for me they are very small and I wanted to increase them in size

**Comment by johnfactotum** [comment, marked as answer]
You can't. The size is hardcoded.

Actually I don't know if it's possible to add the ability to resize for GtkPopovers. At least I've never seen any app do this.

An alternative would be to add some heuristic to automatically increase the size of the popover depending on the length of the text. That would probably be the easier and better solution.

**Reply by i-m-a-r** [reply]
Ok, thanks
